CHANGELOG.md

```diff
@@ -4,9 +4,56 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 
 ## Next
 
+## 0.17.1 (2024/08/21)
+
+### New and Improved
+
+* Add `GetDownstreamWorkersTimeout` config option which represents the period of time (as a duration) timeout
+  for GetDownstreamWorkers call in DownstreamWorkerTicker. This is currently not documented and considered internal.
+  ([PR](https://github.com/hashicorp/boundary/pull/5007))
+
 ### Bug Fixes
 
-* fix(minio): disable multipart uploads to allow session recordings up to 5GB
+* Minio large file support: Disable multipart uploads via minio to fix an issue where the file checksum is set incorrectly on each part 
```

Comment:
I had a quick sync with Jeff and it is proposed that instead of duplicating all the Bug Fixes here rather just add a one liner like:
`Contains Bug Fixes from 0.16.3` I am not sure if we want to include the Security changes in there as well, but it is strange reading a changelog that contains duplicate entries moving from 0.16.3 - 0.17.1
@psekar @AdamBouhmad thoughts?

Reply:
I think it makes to have one entry in the latest verison 0.17.1 but mark it as backported to 0.16.3 Enterprise and Community editions. Probably this is the way forward for other backport releases too.

Reply:
Yeah that's better. I for security changes, we could do the same. Something similar like `Contains Security Fixes from 0.16.3` under the `Security` heading